docs/static-s3-website.md

```diff
@@ -0,0 +1,78 @@
+## Construct StaticS3Website
+
+This construct creates an S3 bucket and upload any content into it.
+It is a composition of the following CDK constructs:
+    - [AWS S3 Bucket lib](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-s3)
+    - [AWS S3 Bucket deployment lib](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-s3-deployment)
+
+Why to use it over the CDK constructs?
+    - It has good defaults. e.g., enforcing ssl; restricted access; auto deleting bucket on deleting the stack.
+    - easier interfaces. i.e. less configuration
+    - Unified construct
+
+### Usage
+
+#### Simple S3 static website
+
+```ts
+    // This will create the bucket `my-website-bucket` and upload the files in your local `dist` folder into it
+    new StaticS3Website(stack, 'my-website', {
+        bucketName: 'my-website-bucket',
```

Comment:
Do the bucket name defaults to the id of the construct or it's a required prop?

Reply:
I'd rather have it as a required prop, so it be more explicit when building/reading the construct.
But having it as default is indeed an option.